daprdocs/content/en/reference/components-reference/supported-bindings/cfqueues.md

```diff
@@ -0,0 +1,236 @@
+
+{{< /tabs >}}
+
+## Generate an Ed25519 key pair
+
+All Cloudflare Workers listen on the public Internet, so Dapr needs to use additional authentication and data protection measures to ensure that no other person or application can communicate with your Worker (and thus, with your Cloudflare Queue). These include industry-standard measures such as:
+
+- All requests made by Dapr to the Worker are authenticated via a bearer token (technically, a JWT) which is signed with an Ed25519 key. 
+- All communications between Dapr and your Worker happen over an encrypted connection, using TLS (HTTPS).
+- The bearer token is generated on each request and is valid for a brief period of time only (currently, one minute).
+
+To let Dapr issue bearer tokens, and have your Worker validate them, you will need to generate a new Ed25519 key pair. Here are examples of generating the key pair using OpenSSL or the step CLI.
+
+{{< tabs "Generate with OpenSSL" "Generate with the step CLI" >}}
+
+{{% codetab %}}
+<!-- Generate with OpenSSL -->
+
+> Support for generating Ed25519 keys is available since OpenSSL 1.1.0, so the commands below will not work if you're using an older version of OpenSSL.
+
+> Note for Mac users: on macOS, the "openssl" binary that is shipped by Apple is actually based on LibreSSL, which as of writing doesn't support Ed25519 keys. If you're using macOS, either use the step CLI, or install OpenSSL 3.0 from Homebrew using `brew install openssl@3` then replacing `openssl` in the commands below with `$(brew --prefix)/opt/openssl@3/bin/openssl`.
+
+You can generate a new Ed25519 key pair with OpenSSL using:
+
+```sh
+openssl genpkey -algorithm ed25519 -out private.pem
+openssl pkey -in private.pem -pubout -out public.pem
+
+# On Mac, using openssl@3 from Homebrew:
+$(brew --prefix)/opt/openssl@3/bin/openssl genpkey -algorithm ed25519 -out private.pem
+$(brew --prefix)/opt/openssl@3/bin/openssl pkey -in private.pem -pubout -out public.pem
+```
+
+{{% /codetab %}}
+
+{{% codetab %}}
+<!-- Generate with the step CLI -->
+
+If you don't have the step CLI already, install it following the [official instructions](https://smallstep.com/docs/step-cli/installation).
+
+Next, you can generate a new Ed25519 key pair with the step CLI using:
+
+```sh
+step crypto keypair \
+  public.pem private.pem \
+  --kty OKP --curve Ed25519 \
+  --insecure --no-password
+```
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
+Regardless of how you generated your key pair, with the instructions above you'll have two files:
+
+- `private.pem` contains the private part of the key: use the contents of this file for the **`key`** property of the component's metadata.
+- `public.pem` contains the public part of the key, which you'll need only if you're deploying a Worker manually (as per the instructions in the previoius section).
+
+{{% alert title="Warning" color="warning" %}}
+Protect the private part of your key and treat it as a secret value!
+{{% /alert %}}
+
+## Related links
```

Comment:
Is it worth referencing any CloudFlare queues documentation here? It generally helps and is recommended.

Reply:
Done